### PR TITLE
Compat data for navigator.windowControlsOverlay API

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary
Over in https://github.com/mdn/content/issues/10539 I'm proposing to add MDN documentation for the new Window Controls Overlay PWA feature. One part of this feature is the `navigator.windowControlsOverlay` API. So in order to document it in MDN, we need its compat data here on this repo.

#### Test results and supporting details
The Window Controls Overlay feature is documented here: https://web.dev/window-controls-overlay/ and here https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay
and its current spec is on the WICG repo here: https://wicg.github.io/window-controls-overlay/
It was implemented in Chromium in 93 (available as origin trial then).

### CODE REVIEW

**Comment by captainbrosset** [comment]
Regarding the version number: the implementation made it in chromium 93, but it was only available behind a flag, or as an original trial then.
It will ship for all users in 97 or 98. Let me know which version should be used in this BCD entry.

**Comment by queengooborg** [comment]
Hey @captainbrosset, thanks for your PR!  Since this will be shipping in Chrome 97 (as per [ChromeStatus](https://www.chromestatus.com/feature/5741247866077184)), we should be using that as the version number.  (Since origin trials are based upon time, rather than versions, there's no good way to represent them in BCD -- we've also been asked not to include them in our data anyways.)

**Comment by captainbrosset** [comment]
@queengooborg I've updated the version number.

File: api/Navigator.json [hunk 1]
```diff
@@ -4289,6 +4289,54 @@
...
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (queengooborg) [inline api/Navigator.json:4335]:
```suggestion
            "standard_track": true,
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for the quick changes!  There's just one last change to make, and then we should be good to go!

**Comment by captainbrosset** [comment]
@queengooborg I made the change as requested. So we should be good to go!

I'm curious about the `standard_track` meaning. I found [this](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#status-information):

> If standard_track is true, then the feature is part of an active specification or specification process.

In this case, the spec is an WICG unofficial draft. Does this still quality as "standard"?

**Comment by queengooborg** [comment]
This is a great question, I'm glad you asked!  Unfortunately, this question has a different answer based upon who you ask: some will say that WICG doesn't count as standard track, others (like me) do.  Other times the draft's status (Track Release, Working Draft, etc.) make a difference in one's judgement.

The way I've been choosing whether to mark something as standard track is:
- Is it a spec listed in w3c/browser-specs (AKA, will the linter take it)?
  - If not, *should* the linter accept it (W3C official draft, open PR/issue for w3c/browser-specs, etc.)?
- Is this feature in the most recent working draft of the spec?

**Review by queengooborg — APPROVED** [review]
This is looking good to me, thank you for the quick changes!

**Comment by captainbrosset** [comment]
> * Is it a spec listed in w3c/browser-specs (AKA, will the linter take it)?

Thanks a lot for your insight on this. Your way of going about it makes a lot of sense to me, and this particular wicg spec was added recently: https://github.com/w3c/browser-specs/commit/8d10b54a398acde53d9a1f5202946312cf38fb2c

**Pull request merged**